.github/workflows/release.yaml

```diff
@@ -5,6 +5,10 @@ on:
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+"
 
+permissions:
+  id-token: write
+  contents: read
```

Comment:
To get `ID token` from GitHub OIDC Provider, we need to add permission configuration (allow) `id-token: write`.
https://docs.github.com/ja/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-cloud-providers

Also, if we add `permissions` in the YAML file, the default permission `contents: read` is NOT allowed.
So, in this case, we need to set (allow) `contents: read` explicitly.
The `contents: read` is required to checkout this repository in the CI (it's required for `actions/checkout@v2`).